k8s/production/monitoring/kubernetes-event-exporter/sealed-secrets.yaml

```diff
@@ -5,7 +5,7 @@ metadata:
+    config.yaml: AgCjWTVhObBZhIg/RtuSNZu7kzBKrFX8METb/gNZnT18j5SDYoLOWI6JIHJ4R6LLYN0wcAfiUpqkFjAVcqiD4a5t/oCAGR0OfSZQ1sSGhhexY0NLwzzQVuGyaIvqd1PB+r9By8uPCXJWLR9W6oPOlqsXOoj+IDqeYcp1GlKR7ZB+BVvD/Mzy7y6baBnsYKF6WXPuZanb3VRXqqe91gb0+maSLWQlEXj4v2wMvxrCMqzvMTJMZLVn0AfJXrsdBLl6EN/lEuzmKuv28MBIhF14AiMHGhxsCtaaWOLQ/tJK4LNQdl4s0FjFm/Hte5JNGK3kL0NFsqnxA6aqeorzIlB9iotMlcBvhDygNRlTz1GyyKS9kIf0cV3W7YJoDoQAy7mccfsPZPQ/UXvGWUqIuz+WKaAXyrWvu2eCC5MoxC6KXQOfYpIfteTQhED0Uprlly5SJ840Mazs6D4x+S5VlrWpoxhkP4GESkI7KwPAGTBnYU/8aow7rzpkGA5wlEngYLR758Fr53eai5FIVrWeWF4+VwDjLnU/M5L5JY7F0jOW+QGErJBmfdY/pTAp2HArR3a7dpvNXv83ns9XrQWHha7+GcgPqnuccs+HtG9DrSezbi9xVJhZTnpIPFBa+2hAJj5vs1gn5siN+kWZA9Y1u5mDu6DdkdcYMxtM/V2fN+AXMynpT+tJsrJhbAbOXfUOnT/PQFq2DhfQ1Y/0WANMfAFtpQOnkfYsYrLLltj2z5HfJLpYHjXNLMgex1Dhbc3O4vrn838fI1A+S3zD8/ELGcKyoMJoOTfhmdonm2zZEOlNxL6GU2zhTQY4HiRN9lYri+ZbOhnwZkKpPJZPCz+QijRWSLeVK+3gr6ZiKLbEE5FCTzGNphge7/N3XgcM/0KbqUydOBHW7lTRlQrQpi4JMfba+FFWSVA4U1g4knYhutwDMXivJKIjZ364kh/LCwrKOPu4HPBBp8OmGCu7ceujJLbyaE0LAafFMwPC1fH0vMJCyBYJvYlxV0uuzt93hBnltmeX2buvS7iojZE1oY+SNAe8jSAZJyKYoVsQuKF557Z82juNNlHLnhy5gzcPh4RBc4Ww7xzSUb7jT24MVjMxI0pbY/wq9PTmpQgXGFuPRbaUCCjZOHJGkDi6vXYGwl31coJYI8+CojkM8i84oXuVqicPuCDqpNDWAlMWYfVeiHtivMEjeOnXWDwQnv+AQ/1h2HqKPAzny7AIT5WvYYZc7BPGIoPe/6WgLjhBBWwsm3N0NkPrx1uSwCXqKpaJwXtFFnDY5wjlO89LWfvC3Sd8EwyFscuAX4nSgGc7gQ9ONXT3LRU2WLvAMAXSx+J/aNTMKGQZobBZn9OHjNLKK89O8kzppC21E5rzNnb+QWO0xeMHpwYw03tuCe2NWBxlnAB/eWyrzmSP6c6SWXivRw21biNR75lIe5URHjM0hPRZlpDv4kXJmiAv9nL9EExk9agxnVgHRyDHwBv2s/n5QqCbpHkvL8LsPLczz+EMko2VIdv0mAyy/I74H1Yawlw3+t68BbmBNDdBoOqH2Hj4F1uxGan4jdV8sjtw01V9Mi5fcra9CZHtKNUYgt3zEMR5bjqz5Fv0iTROCxohOqAzRjrL1TzFCmgnOiBHLZadB06f0QpMA/ZP+I1SPl1Dm3v5AQR3gZ8Nb3VdpgYgVzmLydNva2p/hXGHW4JkfjZMInPtds/WsLtw0lijc5ECrlOnY3H+RcsV/ABCkD+8yH9BkAC/zRjT/BZJtb84NQ80o8tOahvaTxvm7LQLyFkDzKFmDEKuYw0loaPEIjU+kcoOejcsx8GwYbIGaQU=
```

Comment:
I re-encoded the change to 3 and 15 for the kubeQPS/kubeBurst fields, along with the newline fix.
```suggestion
    config.yaml: AgAk+XxVILbn3MRI7jmjAM+kKIsN9FYIH/vGUqXMX1Z6AIEubtPa1FXm3AvzjyML75noEqFDYmgjWsA3pAYqI5InLMxjmTFDewuQsv/Q6JkZk/ciQRn9BldUAxlC6jG7Xmu1fsoLJ/DQBf7E8CFDEd91AuW8sIw0gXZPVitgQyck/Z6tp6Pg4/uTxl2yByNL97Bzckr81Fo+74lACg5Pz9vPgr41LLaoGW22AJqLUFka4f5g9Y7l0c0CRljzH9j7z2iA1i9kUKXfdZWlr7A2DeuTvFfSKr0NY59EhKlehBc5HFXrEAq2+mw0hvX4t5Zv3BM3qddG7s9s6xgMJjJNEE7Gt2qUawBO0wq7ZS3DzeGmB6StIASdLm7RMv5TlDeRxFjFOvrszaxj3q0keaTvmrywDWRlR7tGWGYZPF8AiwIqevvIpb3maIw+6y5AvJxrkyg2MwsxU+0cViF55xOERR0NUDLPqiXatgz91ArABWQr8VPMBsti1C2K102DZLxVRwBkavU93+ytnZVQdzEVwECFmpup4H7mhqVTe65MKVUzTXJdYBr3PpC8mhzYlGnG2FJjkWBpEamq4uPU7aKHzrI3E7rQ9fw4yH1KZ9kD2LFZwMD7b4GqvL1Bjw6hoRtHWKSjA5e1pvLUVwamu4bkN/ftFQjnWVaUWBnIw46b9Js+PCbsCazeKHKusZlkyWnO2n+dDEmv6ZHMZJCOt3JTo06+Lt4D0gls6jK04s/XfxAX3hRbtem5mkSjxLAsLsCk0bcHuEix4IZkecb7Ts6X0ZejJffW0X9J7J+cRF1hgsxy5cc6C1zRDjnTMJst7EmAPQATh7g/COBKC1l6a7edjK36Ib6g6JhULmlXI4eQvIwE8SACCiY4YRD38tPYKTBm7i7W7FJe6GBm0C9+lFETGWimuwxLV4VbXzSwvSnx4dqm4bH2pwzbw7Br8vyvbzRar0UerUriNW+twDgqQO2Trl6aDwyanLcqU3yrcZagBTrJUG5P/UmIFfgmo7nQ/bt4oq/Ip+t9ZMwOPzpL9zMB1uV87qxorstwB/Wfe4+AV62S0p0lRKbBOgIsz5x3nsulncj9eDgq9tV8XgI0lA1GPpEPukOp2VNzYdQEgcBBgNV+oZbIbA1CwIVY84KdGxmtVDOTarLxhJIKVMoFn5KUj7aGj89MpEHHGYCjEUoJ25pMZ6nIQ9MdkBu6iIXV8ffmdC8/da1sdpT4p+LFCMpAD1DY50B8GJ0B5u+sBHuTda5/MkiRwD0zt13EQatwwj3LWzVq5kI0VbpYjMkkgZM97FsjB7Q6N9Bbol4su3E8kAShOarlgvSA/PG3hUpFkOVsCvmh/++EtLYqnlNqri0Zsnn2MymkVoaZaRHMckoXKnpSSL/UNObaI0qTB7n7dUl9+dpy109WLxfV/GxFeafaTB+b12I6VYHO68Qx+3oprSAMjB5VQCbVBZFTK1thhojuDJNBCwF+g4Xg2vs77KZ9klHEsadDwhSqsd914ramWnBkHnxVPngO5jHOn0HPvhvkvHzM0PSR5oot3E1WOWNlpjZdt2C7mXsmA3kDR5i4sYNZUP9RphMZB4XuoQO9lXuzu8dioxJGlUG4a1uGssFIGz+TXJ4KTcD/dOuHuLPA4+kqwj4aGOlnpu4I8unp4wTTJk0/4fLa1RXKyZZuKRV7z/Hs8MvyU/7YzhZlILL3U5Rkl5A62oInTSzbQc2FeH5/3a6xJ3+6wiQxitfCNQe5tz1wZxXXvN50dcNrWZo5hZ5NUMQrISj2VnAnLg==
```